CHANGELOG.md

```diff
@@ -1,5 +1,145 @@
+* Support for constructing a `rustls_server_cert_verifier` that uses the
+  platform operating system's native certificate verification functionality was
+  added. See the [`rustls-platform-verifier`] crate docs for
+  more information on supported platforms.
+    * Use `rustls_platform_server_cert_verifier()` to construct a platform verifier
+      that uses the default crypto provider.
+    * Use `rustls_platform_server_cert_verifier_with_provider()` to construct a 
+      platform verifier that uses the specified `rustls_crypto_provider`.
+    * The returned `rustls_server_cert_verifier` can be used with
+      a `rustls_client_config_builder` with
+      `rustls_client_config_builder_set_server_verifier()`.
+
+* When using `aws-lc-rs` as the crypto provider, NIST P-521 signatures are now
+  supported.
+
+[`rustls-platform-verifier`]: https://github.com/rustls/rustls-platform-verifier
+
+### Changed
+
+* `rustls_server_config_builder_new()`, `rustls_client_config_builder_new()`,
+  `rustls_web_pki_client_cert_verifier_builder_new()`, and
+  `rustls_web_pki_server_cert_verifier_builder_new()`, and
+  `rustls_certified_key_build` functions now use the process
+  default crypto provider instead of being hardcoded to use `*ring*`.
+
+* `rustls_server_config_builder_new_custom()` and
+  `rustls_client_config_builder_new_custom()` no longer take custom
+  ciphersuites as an argument. Instead they requires providing
+  a `rustls_crypto_provider`.
+    * Customizing ciphersuite support is now done at the provider level using
+      `rustls_crypto_provider_builder` and
+      `rustls_crypto_provider_builder_set_cipher_suites()`.
+
+* `rustls_server_config_builder_build()` and
+  `rustls_client_config_builder_build()` now use out-parameters for the
+  `rustls_server_config` or `rustls_client_config`, and return a `rustls_result`. 
+  This allows returning an error if the build operation fails because a suitable
+  crypto provider was not available.
+
+* `rustls_client_config_builder_build()` now returns
+  a `RUSTLS_RESULT_NO_SERVER_CERT_VERIFIER` `rustls_result` error if a server
+  certificate verifier was not set instead of falling back to a verifier that
+  would fail all certificate validation attempts.
+
+* The `NoneVerifier` used if a `rustls_client_config` is constructed by
+  a `rustls_client_config_builder` without a verifier configured has been
+  changed to return an unknown issuer error instead of a bad signature error
+  when asked to verify a server certificate.
+
+* Error specificity for revoked certificates was improved.
+
+### Removed
+
+* The `ALL_CIPHER_SUITES` and `DEFAULT_CIPHER_SUITES` constants and associated
+  functions (`rustls_all_ciphersuites_len()`,
+  `rustls_all_ciphersuites_get_entry()`, `rustls_default_ciphersuites_len()` and
+  `rustls_default_ciphersuites_get_entry()`) have been
+  removed. Ciphersuite support is dictated by the `rustls_crypto_provider`. 
+  * Use `rustls_default_supported_ciphersuites()` to retrive
```

Comment:
```suggestion
  * Use `rustls_default_supported_ciphersuites()` to retrieve
```

Reply:
The only thing worse than a silly typo is a silly typo copy-pasted a bunch of times :upside_down_face: 

Thanks for catching. Fixed throughout.